airflow/operators/bash.py

```diff
@@ -100,12 +100,13 @@ class BashOperator(BaseOperator):
     @apply_defaults
     def __init__(
             self,
+            *,
```

Comment:
Can you explain why? Is this backward compatible?

Reply:
See the linked issue #9942 - it explains whys.